paasta_tools/cli/cmds/secret.py

```diff
@@ -438,23 +470,17 @@ def paasta_secret(args):
         return
 
     if args.action in ["add", "update"]:
+        # this will only be invoked on a devbox only
```

Comment:
nit:
```suggestion
        # this should really only ever be invoked on a devbox
```

(technically, my real nit is just that we say "only" twice here - so feel free to just remove the last "only")